Gruntfile.js

```diff
@@ -263,10 +331,13 @@ module.exports = function (grunt) {
     grunt.registerTask('build:visual-tests', ['check', 'clean:visualTests', 'copy:visualTests', 'assemble:visualTests']);
     grunt.registerTask('build:components', ['check', 'clean:dist', 'version', 'concat:dist', 'uglify:dist', 'concat_css:all', 'cssmin:dist', 'jasmine:test']);
     grunt.registerTask('build', ['build:components', 'build:visual-tests']);
-    grunt.registerTask('dev:serve', ['build', 'connect:dev', 'watch']);
+    grunt.registerTask('dev:serve', ['connect:dev', 'watch:dev']);
     grunt.registerTask('dev', ['build', 'watch']);
     grunt.registerTask('ci', ['default']);
     grunt.registerTask('test', ['jasmine:test', 'build:visual-tests']);
     grunt.registerTask('serve', ['connect:keepalive']);
+    grunt.registerTask('site:dev', ['clean:site', 'copy:site', 'concat:site', 'less:site', 'assemble:site']);
+    grunt.registerTask('site:serve', ['connect:site', 'watch:site']);
+    grunt.registerTask('site', ['site:dev', 'uglify:site']);
```

Comment:
Thats a good idea - tasks could also be added which HTML lint / validate to the assembled output, making the travis build more meaningful.